tests/test_workflow_api.py

```diff
@@ -789,9 +790,15 @@ async def test_prevent_removal(
         workflow2 = await workflow_factory(
             id=workflow.id + 1, project_id=project.id
         )
+        # without the next to lines the test fails on CI but passes locally
```

Comment:
FYI, this is due to the usual lack of full support for foreign-key constraints in sqlite.